manual/creating_a_user_application/index.md

```diff
@@ -31,25 +34,70 @@ With the following code a canvas is drawn (→ see also [Graphics]({{ '/manual/g
    caption="Canvas."
 %}
 
-A standalone program in C++ should be created for this code. 
+A standalone program in C++ should be created with this code.
 
 > **Note**
 >
-> If you use this code in a ROOT macro (→ see [ROOT macros and shared libraries]({{ '/manual/interacting_with_shared_libraries' | relative_url }})), you can only execute it with ROOT.
+> If you use this code in a ROOT macro (→ see [ROOT macros and shared libraries]({{ '/manual/interacting_with_shared_libraries' | relative_url }})),
+> you can only execute it with ROOT.
 
 
-## Quitting ROOT when closing the canvas
+## Generating the PDF output of the canvas and quitting
+
+A standalone program in C++ contains the `main()` function, the starting point for the
+application execution. For this reason, create a C++ file that you can compile.
+
+The first lines of the C++ file include ROOT header files. The names of the ROOT header
+files are almost always the same as the class names (here {% include ref class="TF1" %} and {% include ref class="TCanvas" %}).
+
+{% highlight C++ %}
+#include "TF1.h"
+#include "TCanvas.h"
+
+int main(int argc, char **argv)
+{
+   TCanvas* c = new TCanvas("c", "Something", 0, 0, 800, 600);
+   TF1 *f1 = new TF1("f1","sin(x)", -5, 5);
+   f1->SetLineColor(kBlue+1);
+   f1->SetTitle("My graph;x; sin(x)");
+   f1->Draw();
+   c->Print("demo1.pdf");
+   return 0;
+}
+{% endhighlight %}
+
+Save the code in file, for example as `demo1.cxx`.
+
+Compile the `demo1.cxx` file as follows:
 
-A standalone program in C++ contains the `main()` function, the starting point for the application execution. For this reason, create a header file that you can compile.
+{% highlight C++ %}
+   g++ demo1.cxx $(root-config --glibs --cflags --libs) -o demo1
+{% endhighlight %}
+
+Then you can run the program as follows:
+
+{% highlight C++ %}
+   ./demo1
+{% endhighlight %}
+
+You will get the following output:
+
+{% highlight C++ %}
+Info in <TCanvas::Print>: pdf file demo1.pdf has been created
+{% endhighlight %}
+
+`demo1.pdf` contains the plot of the function `f1`.
 
-The first lines of a header file include ROOT header files. The names of the ROOT header files are almost always the same as the class names (for example, {% include ref class="TApplication" %}, {% include ref class="TF1" %}, {% include ref class="TCanvas" %}).
+## Quitting ROOT when closing the canvas
 
-Use {% include ref class="TApplication" %} to create a ROOT application environment that provides an interface to the windowing system event loops and event handlers. 
 
-To run the canvas as a standalone application you must create a `TApplication` object. Calling the `Run()` method starts the event loop.
+To see the output on screen use {% include ref class="TApplication" %} to create a ROOT application environment that
```

Comment:
Use {% include ref class="TApplication" %} to display the output on a screen. {% include ref class="TApplication" %} creates a ROOT application environment that provides an interface to the windowing system event loops and event handlers.